start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
+                         install <span class="c_snip_inline">pbench-agent</span>. The recommended method for installation is through
+                         ansible playbooks.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         The steps are:
+                       </p>
+                       <div class="info_text" style="text-align:justify">
+                           <li>Make sure that you have ansible and ansible-galaxy installed on your
+                             laptop (or wherever you decide to run the playbooks).</li>
+                           <li>Install the ansible roles from Ansible Galaxy.</li>
+                           <li>Tell ansible where to find these roles.</li>
+                           <li>Create a playbook (you can copy the one below).</li>
+                           <li>Create an inventory file naming the hosts on which you wish to install
+                             <span class="c_snip_inline">pbench-agent</span> and including information about where the config file and ssh key
+                             file can be found.</li>
+                           <li>Run the playbook.</li>
+                       </div>
+                       <p class="info_text"  style="text-align:justify">
+                         The playbook will install the pbench repo and it will then use it to
+                         install the pbench-agent RPM; finally, it will install the
+                         localization pieces (config file and ssh key) that will allow you to
+                         send results to the server.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         The prerequisite assumption is that you can ssh as root without a
+                         password into the relevant set of hosts:
+                       </p>
+		       <p class="c_snip">
+                         ssh-copy-id root@host
+                         ...
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         for each host in your inventory (that includes <span class="c_snip_inline">localhost</span> if you are going to
+                         install <span class="c_snip_inline">pbench-agent</span> on your local host).
+                       </p>
+                     </div>
+                   </div>
+                   <!-- Subsection: Install ansible and ansible galaxy (one-time task)-->
+		   <div class="accordionCard">
+		     <h4 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Install ansible and ansible galaxy (one-time task)</span>
```

Comment:
Capitalize "Ansible" and "Ansible Galaxy"?

Reply:
Fixed.

Reply:
Actually, that looked weird to me: after all we are not installing Galaxy, just the program to access it. So I made them monospace to indicate that we are talking about the programs. Is that OK?